Resources/Prototypes/Entities/Clothing/Ears/headsets.yml

```
@@ -19,6 +19,8 @@
   name: cargo headset
   description: A headset used by the quartermaster and his slaves.
   components:
+  - type: Headset
+    channels: [broadcast, supply]
   - type: Sprite
```

IMO it's much nicer to do these as a list, like so:

```suggestion
  - type: Headset
    channels:
    - broadcast
    - supply
  - type: Sprite
```